command/launch.go

```diff
@@ -64,6 +69,11 @@ func (c *LaunchCommand) Run(args []string) int {
 		flags.Parse(flags.Args()[1:])
 	}
 
+	if arn != "" && roleName != "" {
+		fmt.Fprintln(os.Stderr, "InvalidParameterCombination: The parameter 'roleName' and 'arn' can not be used in combination")
```

Comment:
https://github.com/dtan4/ec2c/pull/10/files#diff-0380a26c6fb593cb2283592d0ba604d4R127

Already display error message. 👀 

Change "panic" to "return 1" will be next PR.